README.md

```diff
@@ -0,0 +1,8 @@
+# mail_room
+401 week1: mail_room assignment
+
+This is an app for generating a thank you letter for donations. 
+
+to run:
+
+$ python src/mailroom.py
```

Comment:
I'd like to see a more complete README file.  More information about what it can do, what you should expect to enter at any point, so on.